windows.md

```diff
@@ -328,7 +328,15 @@ Before you start, please make sure that you're running Windows 10 or Windows 11.
 
     <img src="windows-7-virtualisation-enabled.png"><br><br>
 
-    If virtualization is not enabled, [enable VT-x or SVM in your UEFI or BIOS](https://example.com/add-link-here).
+    Only if virtualization is not enabled, **Enable VT-x or SVM in your UEFI or BIOS**
+
+    1. Reboot your computer
```

Comment:
- [ ] let's use `Restart` instead of `Reboot`, simpler language that more people will be familiar with
- [ ] you should probably explain the part about the black screen and the keys to press earlier before telling people to restart their computer (because they may wait for Windows to completely load before reading further)

Reply:
15293d790179e6b6e588e046092db9308d2b4e74